features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -1873,6 +1873,31 @@
             <Scopes>internal_login</Scopes>
         </Resource>
 
+        <Resource context="(.*)/api/server/v1/organizations" secured="true" http-method="GET">
```

Comment:
Lets remove the empty new line between the configurations

Reply:
The identity configurations to support organization management feature is removed from the identity.xml and indentity.xml.j2 files of the framework repository and added via the deployment.toml configuration at the B2B feature enable setup. (The isb2bconnector) https://github.com/wso2-enterprise/identity-org-mgt-connector/blob/main/modules/distribution/conf/deployment.toml